modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -647,6 +647,143 @@ definition {
 		AppBuilderAdmin.deleteEntry(itemName = "my custom object");
 	}
 
+	@description = "Delete fieldsets used in a form view and when it is not used"
+	@priority = "5"
+	test DeleteFieldSet {
+		property portal.acceptance = "true";
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object one",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		FormViewBuilder.addFieldByDoubleClick(fieldType = "Text");
+
+		FormViewBuilder.validateFieldLabel(fieldLabel = "Text");
+
+		Click(locator1 = "FormViewBuilder#RULE_BUILDER_GO_BACK");
+
+		FormViewBuilder.setUpTwoFieldSets(
+			fieldSetName = "Field Sets",
+			fieldType = "Text",
+			fieldType2 = "Numeric");
+
+		FormViewBuilder.goToFieldsetsTab();
+
+		FormViewBuilder.deleteFieldSetEntry(
+			rowNumber = "1",
+			selectAction = "Delete");
```

Comment:
Hi @nkkwon , after deleting a fieldset, you need to assert that the fieldset is no longer present.